dataframe-jdbc/src/main/kotlin/org/jetbrains/kotlinx/dataframe/io/readJdbc.kt

```diff
@@ -216,6 +216,67 @@ public fun DataFrame.Companion.readSqlQuery(
     }
 }
 
+/**
+ * Converts the result of an SQL query to the DataFrame.
+ * Also, it could be a just name of the SQL table.
+ *
+ * NOTE: It should be a name of one of the existing SQL tables,
+ * or the SQL query should start from SELECT and contain one query for reading data without any manipulation.
+ * It should not contain `;` symbol.
+ *
+ * @param [sqlQueryOrTableName] the SQL query to execute or name of the SQL table.
+ * @param [limit] the maximum number of rows to retrieve from the result of the SQL query execution.
+ * @param [inferNullability] indicates how the column nullability should be inferred.
+ * @return the DataFrame containing the result of the SQL query.
+ */
+public fun DatabaseConfiguration.toDF(
```

Comment:
Actually, it might even be a good idea to get different names per overload:

- `ResultSet.toDataFrame()` makes sense, because we're converting a resultset to a dataframe.
- For `DataBaseConfiguration.toDataFrame()` we aren't converting the configuration to a dataframe, so the name is misleading. We'd better consider something like `DataBaseConfiguration.getDataFrameFor(sqlQueryOrTableName)` or something similar.
- Same for `Connection`

Reply:
The problem is following:
- extract data from Connection via extension function is normal (it's a pipe)
- extract data from DbConfig - weird, but we're calling methods on DataFrame.readSQLTable (we are not reading SQL table from from DataFrame? It's just a place for method)

Different names make it's more and more complicated 

Much more better will be to have something like DatabaseReader.readSqlTable instead of DataFrame.readSqlTable

Also, as an alternative - I could remove extension functions for Connection/DbConfig and stay only with ResultSet

Reply:
Let's live a little bit, if it will be bad on Demos, I will remove